source/elements/oneMKL/source/architecture/other_architecture.inc.rst

```diff
@@ -11,10 +11,41 @@ This section covers all other features in the design of oneMKL architecture.
 
 .. _onemkl_spec_current_version:
 
-Current Version of this oneMKL Specification
+Specification Version and Compliance
 +++++++++++++++++++++++++++++++++++++++++++++
 
-This is the oneMKL specification which is part of the oneAPI specification version 1.0.0.
+Each oneMKL domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
```

Comment:
The oneMKL spec allows for an implementation of the spec to implement just 1 API and throw unsupported for all of the other APIs in that domain. (See https://github.com/uxlfoundation/oneAPI-spec/blob/main/source/elements/oneMKL/source/architecture/api_design.inc.rst)
In that case, is the implementation considered conformant to whatever version of the spec has the APIs that are included in the implementation (albeit mostly unsupported), or is that implementation considered non-compliant?

Reply:
My understanding is that the implementation will still be compliant, because it follows the definitions of the specification to handle errors for unimplemented APIs.

However, the implementation would not be conformant to the standards that the industry would expect backend libraries to be, because of lack of most APIs.
Alternatively, it could be a feature that a specific hardware-software use case only requires or supports certain APIs as per their app's needs.